apps/judicial-system/backend/src/app/formatters/formatters.ts

```diff
@@ -141,19 +141,19 @@ export function formatCourtResubmittedToCourtSmsNotification(
 
 export function formatDefenderResubmittedToCourtEmailNotification(
   formatMessage: FormatMessage,
-  policeCaseNumber: string,
+  policeCaseNumbers: string[],
   overviewUrl: string,
   courtName?: string,
 ) {
   const subject = formatMessage(
     notifications.defenderResubmittedToCourt.subject,
     {
-      policeCaseNumber,
+      policeCaseNumber: policeCaseNumbers.join(', '),
```

Comment:
There are many changes like this one. I decided not to change the parameter name to the contentful string to avoid having to change the string.